### DISCUSSION TITLE
评论/留言功能支持

### DISCUSSION THREAD

**Discussion by OrzMiku** [AlistGo/alist#3947] [Ideas]
可以对某个文件夹开启评论/留言功能，可以对接使用gitment，gitalk，gitter，valine等评论系统。

**Comment by xhwsd** [comment]
README.md 里嵌入应该满足需求

**Reply by ip520011** [reply]
怎么嵌入呢

**Reply by xhwsd** [reply]
README.md 支持 HTML，想嵌入什么都可以

**Reply by xhwsd** [reply]
代码：
[![ppcWwrR.png](https://s1.ax1x.com/2023/03/29/ppcWwrR.png)](https://imgse.com/i/ppcWwrR)

效果：
[![ppcWra6.png](https://s1.ax1x.com/2023/03/29/ppcWra6.png)](https://imgse.com/i/ppcWra6)

**Reply by xhwsd** [reply]
**抛砖引玉~~~~**

**Reply by OrzMiku** [reply]
> README.md 支持 HTML，想嵌入什么都可以

是这样的，技术力足够也可以自己构建前端修改。我的站点也已经加上了留言。只是希望官方能支持这个功能，能在后台直接配置自然最好。

**Reply by xhwsd** [reply]
我感觉 ALIST 既然是网盘聚合工具，就做好网盘功能，并提供可定制扩展性就可以了。

**Reply by 13095605502** [reply]
请问用的是什么呢，我用leancloud一直不行，不知道哪里错了
